src/code/m_submenu.c

```diff
@@ -174,22 +174,20 @@ void* mSM_ovlptr_dllcnv(void* vram, mSM* submenu) {
     return (void*)((uintptr_t)vram + area->relocationDiff);
 }
 
-#ifdef NON_MATCHING
-// regalloc
 void func_800C47B4_jp(UNUSED void* arg0, UNUSED void* arg1) {
     void* temp_s0;
     Player* temp_v0_3;
     UNUSED void* temp_v0_2;
-    struct_800418D8* temp_v0;
+    OSThread* temp_v0;
     u32 temp_a3; // sp34
     u32 var_t1;  // sp30
     u32 var_t0;  // sp2C
     size_t ovlSize;
     void* temp_a2; // sp20
 
-    temp_v0 = B_800418D8_jp;
-    var_t0 = (temp_v0 != NULL) ? temp_v0->unk_11C : 0;
-    var_t1 = (temp_v0 != NULL) ? temp_v0->unk_104 : 0;
+    temp_v0 = gFaultMgr.faultedThread;
```

Comment:
The regalloc fixed itself by using the correct symbol? lol

Reply:
It's probably because `context.pc` and `context.ra` are `u64` not `s32`